GoogleForCreators/web-stories-wp
# Canvas: Show base color / blurhash while resource is loading

## Context



## Summary

Show base color / blurhash based placeholder for element on canvas. 

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/176723782-2d057043-a230-4249-aeac-50ed8662646c.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Create story. 
2. Insert a number of video / image elements. 
3. Save story as draft. 
4. Go to media library. 
5. Delete video / image used in elements. 
6. Edit draft story.
7. See placeholder. 
8. Resized elements, see placeholder change size as well. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10258
